src/doi.jl

```diff
@@ -59,13 +59,27 @@ function Base.show(io::IO, ::MIME"text/html", dois::Array{T} where {T<:AbstractD
     print(io, "</ol>")
 end
 
+"""
+    meta-data structure from API: https://w3id.org/oc/meta/api/v1/metadata/
+"""
+function metadata_template(doi::String)
+    fields = (:publisher, :pub_date, :page, :venue, :issue, :editor, :author, :id, :volume, :title, :type)
+    rj = Dict(f => "\u0000" for f in fields)
```

Comment:
```suggestion
    rj = Dict(f => "" for f in fields)
```
This was the change I had in mind, this should work as well, or is there a specific reason to use `"\u0000"`?

Reply:
This may work, but from my recollection, the `Base.show` methods didn't like all the fields being empty strings.